src/IssueViz.Security.UnitTests/Hotspots/LocalHotspotTests.cs

```diff
@@ -50,6 +50,6 @@ public void Ctor_VisualizationIsNull_Throws()
                 null,
                 null));
 
-        test.Should().Throw<ArgumentNullException>().Which.Message.Should().Contain("visualization");
+        test.Should().Throw<ArgumentNullException>().WithMessage("*visualization*");
```

Comment:
Complains that `WithMessage` should be used, but that means we have to add wildcards to the string to get the same effect as `Contain`...